src/main/kotlin/no/nav/familie/dokument/storage/ProtectedStorageController.kt

```diff
@@ -0,0 +1,31 @@
+package no.nav.familie.dokument.storage
+
+import no.nav.familie.dokument.storage.attachment.AttachmentStorage
+import no.nav.security.token.support.core.api.ProtectedWithClaims
+import no.nav.security.token.support.core.context.TokenValidationContextHolder
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.beans.factory.annotation.Value
+import org.springframework.http.MediaType
+import org.springframework.web.bind.annotation.*
+import org.springframework.web.multipart.MultipartFile
+
+@RestController
+@RequestMapping("api/mapper")
```

Comment:
Hvorfor har du endret url fra api/vedlegg til api/mapper? Ser ut som kontrolleren legger til og henter vedlegg, som før? :)

Reply:
Var en tanke om at vi også skal bruke den til å lagre pågående søknader (json). Da blir "vedlegg" litt feil.

Så en url'en av type "nav.no/familie/mellomlagring/api/mapper/<mappe>/<dokument-id>". Og der "mappe" peker til konfigurert bucket. MEN svært åpen for navnediskusjonen der :)